packages/ckeditor5-table/src/tableselection.js

```diff
@@ -425,39 +459,30 @@ export default class TableSelection extends Plugin {
 		const startColumn = Math.min( startLocation.column, endLocation.column );
 		const endColumn = Math.max( startLocation.column, endLocation.column );
 
-		const cells = [];
+		const selectionMap = new Array( endRow - startRow + 1 ).fill( null ).map( () => [] );
```

Comment:
`fill( [] )` ?

Reply:
That would make the same array instance for all rows.